assets/js/pages/scheduled.js

```diff
@@ -396,12 +432,30 @@ module.exports = function (name) {
         return calendarArticles;
     }
 
-
-    return {
+    var schd = {
```

Comment:
Could this be given a better name? Coming across references to it elsewhere in the file, it's not clear what it is.